syntax = "proto3";

package math;

service Arithmetic {
  rpc Sum (SumRequest) returns (SumResponse) {}
}

message SumRequest {
  int32 operand1 = 1;
  int32 operand2 = 2;
}

message SumResponse {
  int32 result = 1;
}
